doc/contributing/docs/style.rst

```diff
@@ -21,4 +21,19 @@ Wrong usage: "Replication allows multiple Tarantool *servers* to work on copies
 of the same databases."
 
 Correct usage: "Replication allows multiple Tarantool *instances* to work on
-copies of the same databases."
\ No newline at end of file
+copies of the same databases."
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Don't use i.e. and e.g.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Don't use the following contractions:
+
+*   `"i.e." <https://www.merriam-webster.com/dictionary/i.e.>`_
+    —from Latin "id est". Use "that is" or "which means" instead.
+*   `"e.g." <https://www.merriam-webster.com/dictionary/e.g.>`_
+    —from Latin "exempli gratia". Use "for example" instead.
```

Comment:
```suggestion
    —from Latin "exempli gratia". Use "for example" or "for instance" instead.
```

Reply:
This is a great advice in general, thank you. But I would object to using it in our docs because "instance" is a term that we use a lot. It's best if we reserve it for cases where it means a Tarantool instance.